.github/workflows/NeedsDocumentation.yml

```diff
@@ -3,13 +3,13 @@ on:
   issues:
     types:
       - labeled
-  pull_request:
+  pull_request_target:
     types:
       - labeled
 
 env:
   GH_TOKEN: ${{ secrets.DUCKDBLABS_BOT_TOKEN }}
-  TITLE_PREFIX: "duckdb/#${{ github.event.issue.number }}]"
+  TITLE_PREFIX: "[duckdb/#${{ github.event.issue.number }}]"
   PUBLIC_ISSUE_TITLE: ${{ github.event.issue.title }}
```

Comment:
These lines still need to be fixed for pull requests I believe (currently they're just using the issue key)

Reply:
Good point, I added a fix and tested it in a private repo. It turns out that the PR number of the `pull_request_target` event is available under `pull_request.number`.